src/form/sections/HeaderFooterMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button } from '@folio/stripes/components';
```

Comment:
usually we create a folder for a component/hook/etc to shorten list in folder (where some component related utils/tests/styles are placed)